packages/nimble-components/src/utilities/unit-format/decimal-unit-format.ts

```diff
@@ -36,18 +35,21 @@ export class DecimalUnitFormat extends UnitFormat<DecimalUnitFormatOptions> {
         const intlNumberFormatOptions = {
             maximumFractionDigits,
             minimumFractionDigits,
-            useGrouping: true
+            signDisplay: 'negative'
         };
         for (const scaledUnit of unitScale.supportedScaledUnits) {
             this.scaledUnitFormatters.set(
                 scaledUnit.scaleFactor,
                 scaledUnit.scaledUnitFormatFactory({
                     locale,
-                    intlNumberFormatOptions
+                    // Hack to avoid a ts error about signDisplay not accepting the value 'negative'.
+                    // The value has been supported by browsers since 8/23, but TypeScript still hasn't
+                    // added it to the type definitions. See https://github.com/microsoft/TypeScript/issues/56269
+                    intlNumberFormatOptions:
+                        intlNumberFormatOptions as Intl.NumberFormatOptions
```

Comment:
This cast is a little concerning to me because it would hide any incorrect configuration of `intlNumberFormatOptions`, not just the unsupported `signDisplay` option.

Did you consider any other ways of writing this that would scope the cast to just the type that needs it?

One idea would be to do something like:
```ts
type SignDisplay = Intl.NumberFormatOptions['signDisplay'];

export class DecimalUnitFormat {
    ...

    const intlNumberFormatOptions = {
        maximumFractionDigits,
        minimumFractionDigits,
        signDisplay: 'negative' as SignDisplay
    };
}
```

Reply:
Love it. Thanks! Also did similar things in `DefaultUnitFormat`.